cmd.go

```diff
@@ -0,0 +1,74 @@
+package main
+
+import (
+	"embed"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/psiemens/sconfig"
+	"github.com/spf13/cobra"
+)
+
+//This file is boilerplate to set up a CLI application in go using cobra/sconfig and embed to embed files into the binary
+
+//we embedd the flow.json  and the transactions we need
+//go:embed transactions/award_manually_many.cdc
+//go:embed transactions/adminAddKeys.cdc
+//go:embed flow.json
+var path embed.FS
+
+//For config we use psiemens brilliant sconfig repo, IMHO a lot easier to use then plain viper/pflags
+type Config struct {
+	File       string `default:"recipients.csv" flag:"file,f" info:"Path to file of recipients, one address per line"`
+	BatchSize  int    `default:"100" flag:"batchSize,b" info:"How many floats to award in a single batch"`
+	Workers    int    `default:"100" flag:"workers,w" info:"Workers to paralell mint as"`
+	EventID    uint64 `required:"need to set eventId" flag:"event,e" info:"Id of event to award"`
+	Host       string `flag:"host" info:"Host to mint for if not main"`
+	PrivateKey string `required:"need to set privateKey" flag:"privateKey,k" info:"privateKey to mint as"`
+	Address    string `required:"need to set address" flag:"address,a" info:"Address to mint as"`
```

Comment:
Sorry, this isn't documented in the `sconfig` repo, but `required` needs to be a boolean (i.e. either `required:"true"`, `required:"false"`. Omitting it defaults to false.

Unfortunately it doesn't yet support custom error messages for required fields. Sorry :(

Reply:
Thanks